broker-daemon/index.js

```diff
@@ -234,6 +245,26 @@ class BrokerDaemon {
     this.orderbooks.set(marketName, new Orderbook(marketName, this.relayer, this.store.sublevel(marketName), this.logger))
     return this.orderbooks.get(marketName).initialize()
   }
+
+  /**
+   * Validates engines
+   * We do not await this function because we want the validations to run in the background.
+   * It can take time for the engines to be ready, so we use exponential backoff to retry validation
+   * for a period of time, until it is either successful or there is actually something wrong.
+   * @returns {<void>}
+   */
+  async validateEngines () {
+    this.engines.forEach(async (engine, symbol) => {
+      try {
+        await exponentialBackoff(() => { engine.validateNodeConfig() }, EXPONENTIAL_BACKOFF_ATTEMPTS, EXPONENTIAL_BACKOFF_DELAY, {symbol})
+      } catch (e) {
+        this.logger.error(`Failed to validate engine for ${symbol}, error: ${e}`)
```

Comment:
Do we just want to error here? Or do we need to take some other action (like crash the process)?